ResearchKit/ActiveTasks/ORKAudioStep.m

```diff
@@ -48,6 +48,7 @@ - (instancetype)initWithIdentifier:(NSString *)identifier {
     self = [super initWithIdentifier:identifier];
     if (self) {
         self.shouldShowDefaultTimer = NO;
+        self.shouldStartTimerAutomatically = YES;
```

Comment:
@umerkhan-apple will this make shouldStartTimerAutomatically to YES as default ?